src/main/java/com/gascalc/GasCalcController.java

```diff
@@ -9,9 +9,25 @@
 import com.gascalc.model.GasCalcModel;
 
 @Controller
-
 public class GasCalcController {
 
+	/**
```

Comment:
A lot of documentation via comments was missing within the application, so I added all of the ones for controller methods to further elaborate on what the method takes in and returns.